syntax = "proto3";

message TransactionRequest {
  string sender_account_number = 1;
  string sender_account_type = 2;
  string receiver_account_number = 3;
  string receiver_account_type = 4;
  double amount = 5;
  string reason= 6;
}

message TransactionResponse {
  bool success = 1;
  string message = 2;
}

service TransactionService {
  rpc SendMoney(TransactionRequest) returns (TransactionResponse);
}
